packages/react-mutation-mapper/src/component/oncokb/ReferenceList.tsx

```diff
@@ -7,15 +7,34 @@ import ArticleAbstractItem from './ArticleAbstractItem';
 import PmidItem from './PmidItem';
 
 import styles from './listGroupItem.module.scss';
+import { observer } from 'mobx-react';
+import { computed } from 'mobx';
```

Comment:
again very minor. can we move these to the topmost import block, since these are external dependecies?